packages/liveblocks-core/src/__tests__/client.node.test.ts

```diff
@@ -29,12 +29,12 @@ function atobPolyfillMock(data: string): string {
 
 function enterAndLeave(options: ClientOptions) {
   const client = createClient(options);
-  client.enter("room", { initialPresence: {} });
+  const { room: _, leave } = client.enterRoom("room", { initialPresence: {} });
```

Comment:
not sure if this is a big concern, but we're replacing the old methods here with the new ones, which leaves the old ones untested. Should we leave a legacy test in place until we're ready to remove them?

Reply:
Great point! I realized this new behavior wasn't tests in unit test form yet, so I added a test and added a similar test for the old API in 81edfb325bcc7e6f97f01adee4b62a4faf4f471f. Thanks for catching that!